IanGRap/Squish
# Sprint 1 demo

Contributors: Ian plus team for playtesting

Goal: Create a demo indicative of progress at the end of sprint 1

Important files changed:

BlockPusher.cs: I added a variable for the push distance of a block and removed variables that were not being used.

How to Test:

Load up the Sprint 1 scene under demo. You will want to controllers to have the smoothest play testing experience. Face buttons are jump, trigger buttons are dash and start reloads the scene.

Notes:

The CameraControl and GameManager scripts are temp files and have not been styled. They should be completely rewritten as a new task.
